modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -1017,6 +1017,35 @@ definition {
 			locator1 = "MyWorkflowTasks#MY_WORKFLOW_TASKS_SEARCH_RESULT_BAR");
 	}
 
+	@description = "LPS-172352 - Verify if it is possible to specify the recipient type in a workflow task"
+	@priority = 4
+	test CanSpecifyTheRecipientTypeInWorkflowTask {
+		task ("Given that a user accesses the workflow") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Workflow",
+				panel = "Applications",
+				portlet = "Process Builder");
+
+			Workflow.gotoEditWorkflowDefinition(workflowDefinitionTitle = "Single Approver");
+		}
+
+		task ("When changing the notification type") {
```

Comment:
@thaiscabrals can we update this task description to specify that it is the recipient type you are changing and not notification type?